sites/feedback/feedback-fr.html

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
Please apply the similar equivalent change as requested for the English alternate language

Reply:
Changed. Thanks